cdc/processor/sinkmanager/table_sink_wrapper.go

```diff
@@ -89,6 +94,12 @@ func (t *tableSinkWrapper) updateReceivedSorterResolvedTs(ts model.Ts) {
 	t.receivedSorterResolvedTs.Store(ts)
 }
 
+func (t *tableSinkWrapper) updateReceivedSorterCommitTs(ts model.Ts) {
+	if ts > t.receivedSorterCommitTs.Load() {
```

Comment:
Seems if redo is enabled, both redo_log_worker and table_sink_worker will call this. I think if redo is enabled, only redo_log_worker needs to call this.